ddtrace/utils/version.py

```diff
@@ -0,0 +1,26 @@
+import typing
+
+import packaging.version
+
+
+def parse_version(version):
+    # type: (str) -> typing.Tuple[int, int, int]
+    """Convert a version string to a tuple of (major, minor, micro)"""
```

Comment:
Do we maybe want to call this parse_semver_version? It probably won't work for calver and other versioning strategies

Reply:
should work fine on calver `2020.6.16` -> `(2020, 6, 16)`

Reply:
:+1: might be worth mentioning in the docstring what the behaviour is if there isn't a minor or micro